src/script/common/c_converter.cpp

```diff
@@ -326,7 +326,7 @@ bool is_color_table(lua_State *L, int index)
 
 aabb3f read_aabb3f(lua_State *L, int index, f32 scale)
 {
-	aabb3f box;
+	aabb3f box{-1.0f, -1.0f, -1.0f, 1.0f, 1.0f, 1.0f};
```

Comment:
I would suggest adding a warning if this unexpected default ends up being used.

We could also error immediately (this is not documented) but that might be too harsh.